wa/workloads/pcmark/uiauto/app/src/main/java/com/arm/wa/uiauto/pcmark/UiAutomation.java

```diff
@@ -47,6 +47,7 @@ public void initialize(){
     @Test
     public void setup() throws Exception{
         dismissAndroidVersionPopup();
+        clearPopups();
```

Comment:
If the version check is performed by the OS I'm guessing this will start affecting other applications as well? 
If so would it be possible to updated the existing `dismissAndroidVersionPopup()` method instead of an additional check here so that other applications can benefit as well?

Reply:
We've only seen it on the PCMark application so far. Otherwise this would be something I'd have added there as you suggest.

Reply:
Ok, if this does turn out to be the case we can move it across then.